### PR TITLE
Add demo example

### PR DESCRIPTION
**Pull request by 13shivam**
- removed old demo example
- added new example with span integration

### CODE REVIEW

File: src/Trace/Integrations/Sqs.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+namespace OpenCensus\Trace\Integrations;
+
+class Sqs implements IntegrationInterface
+{
+
+    public static function load()
+    {
+        if (!extension_loaded('opencensus')) {
+            trigger_error('opencensus extension required to load Sqs integrations.', E_USER_WARNING);
```
Comment (Sunn-y-Arora) [inline src/Trace/Integrations/Sqs.php:11]: return if ext not loaded

File: src/Trace/Integrations/Sqs.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                return [
+                    'attributes' => $internalFields,
+                    'kind' => 'client',
+                    'name' => 'Sqs ' . $args->getName(),
+                    'sameProcessAsParentSpan' => false
+                ];
+            }
+        });
+
+        opencensus_trace_method('Aws\Sqs\SqsClient', '__construct', function ($command, $args) {
```
Comment (Sunn-y-Arora) [inline src/Trace/Integrations/Sqs.php]: add unit test for the integration added pls.

**Review by Sunn-y-Arora — CHANGES_REQUESTED** [review]
assuming demo app working, not reviewing that .

add unit test for the integration added

File: src/Trace/Integrations/Sqs.php (hunk 1 above)
Comment (13shivam) [reply src/Trace/Integrations/Sqs.php:11]: added

File: src/Trace/Integrations/Sqs.php (hunk 2 above)
Comment (13shivam) [reply src/Trace/Integrations/Sqs.php]: added

**Comment by Sunn-y-Arora** [comment]
@13shivam looks good, can u check why the builds are failing, we can merge after that

**Review by Sunn-y-Arora — APPROVED** [review]

**Pull request merged**